docs/UserGuide.md

```diff
@@ -76,10 +76,7 @@ If you are an **experienced user**:
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+  e.g `INDEX [n/NAME] [p/PHONE]` can be used as `1 n/John Doe p/97328601` or as `1 n/John Doe`.
```

Comment:
Should we contextualize it with the command find?
so 
e.g `find [i/ID] [n/NAME] [p/PHONE] [e/EMAIL] [d/DEPARTMENT] [r/ROLE] [s/SALARY_LOWER_BOUND - SALARY_UPPER_BOUND] ` can be used as `find n/John Doe` or as `find d/finance`.